AllReadyApp/Web-App/AllReady/Caching/OptimizeRouteResultFromCacheQueryHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using AllReady.Caching;
+using AllReady.Services.Mapping.Routing;
+using MediatR;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace AllReady.Areas.Admin.Features.Itineraries
+{
+    public class OptimizeRouteResultFromCacheQueryHandler : IRequestHandler<OptimizeRouteResultFromCacheQuery, OptimizeRouteResultStatus>
```

Comment:
@stevejgordon, I don't think you're misusing a handler, I think you're mis-naming a handler.

Messages should be named in the language of the business. Technical implementations are a by-product of what we're trying to accomplish. When you think about it, the implementation is a really "technical artifact" of the business operation we're trying to perform.

Up to you in the end, but I'd still suggest `OptimizeRouteResultQuery`